module/VuFind/tests/unit-tests/src/VuFindTest/Recommend/SideFacetsTest.php

```diff
@@ -390,10 +402,17 @@ protected function getMockResults(Params $params = null): Results
         if (null === $params) {
             $params = $this->getMockParams();
         }
+        $options = $this->getMockBuilder(\VuFind\Search\Solr\Options::class)
+            ->disableOriginalConstructor()->getMock();
```

Comment:
Could be simplified:
```suggestion
        $options = $this->createMock(\VuFind\Search\Solr\Options::class);
```

Reply:
Done.